charts/fpl-case-service/values.preview.template.yaml

```diff
@@ -268,9 +261,9 @@ ccd:
       es.data.nodes.url: http://${SERVICE_NAME}-es-master:9200
     extraEnvs:
       - name: DATA_STORE_URL
-        value: jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified
+        value: jdbc:postgresql://fpl-preview.postgres.database.azure.com:5432/{{ .Values.global.databaseNamePrefix }}data-store?sslmode=require&stringtype=unspecified
       - name: DB_URL
-        value: jdbc:postgresql://${SERVICE_NAME}-postgresql:5432/data-store?ssl=disable&stringtype=unspecified
+        value: jdbc:postgresql://fpl-preview.postgres.database.azure.com:5432/{{ .Values.global.databaseNamePrefix }}data-store?sslmode=require&stringtype=unspecified
       - name: DB_USER
         value: hmcts
       - name: DB_PWD
```

Comment:
you will have to pass this `DB_PWD` as secret, this is the same secret you have setup on the server config on [cnp-flux-config](https://github.com/hmcts/cnp-flux-config)

Reply:
this still need fixing?
see example here - https://github.com/hmcts/civil-ccd-definition/pull/3542/files#diff-42fed93979fdbd819455c6d1c5716828f1afd337e1c2980c925e1b9c3cb31e62L46